tedana/metrics/external.py

```diff
@@ -0,0 +1,650 @@
+"""Metrics unrelated to TE-(in)dependence."""
```

Comment:
```suggestion
"""Metrics based on external confounds."""
```

I can foresee adding metrics unrelated to TE dependence that don't rely on external confounds (e.g., kurtosis of the component time series, spatial distribution of the component weights).

Reply:
Changed to """Metrics based on fits of component time series to external time series.""" in my last commit. I didn't want to use the word "confounds" since this can also take task-locked time series.